DFC.App.MatchSkills.Application/Dysac/IDysacSessionReader.cs

```diff
@@ -6,9 +6,9 @@ namespace DFC.App.MatchSkills.Application.Dysac
 {
     public interface IDysacSessionReader
     {
-        Task<DysacServiceResponse> InitiateDysacOnly();
-        Task<DysacServiceResponse> InitiateDysac(DfcUserSession userSession);
+        Task InitiateDysacOnly();
+        Task InitiateDysac(DfcUserSession userSession);
         Task<DysacJobCategory[]> GetDysacJobCategories(string sessionId);
-        Task<DysacServiceResponse> LoadExistingDysacOnlyAssessment(string sessionId);
```

Comment:
Sorry my bad on this one, LoadExisitngDsyanOnly needs to return if it exists. If not then we need to Show the Model error of not a valid session code